Init.ps1

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+<#
+.SYNOPSIS
+    Sets up and initialize the developer Shell Environment for Mobius
+.DESCRIPTION
+    Sets up the proper PATH and ENV to use Posh-Git shell and build environment
+#>
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory = $false)]
+    [switch]$SetupEnv
+)
+
+process
+{
+    $projectName = "Mobius"
+    $latestVs = $null
+
+    function Get-ScriptDirectory
+    {
+        $Invocation = (Get-Variable MyInvocation -Scope 1).Value;
+        if($Invocation.PSScriptRoot -and $Invocation.CommandOrigin -eq "Runspace")
+        {
+            $Invocation.PSScriptRoot;
+        }
+        Elseif($Invocation.MyCommand.Path)
+        {
+            Split-Path $Invocation.MyCommand.Path
+        }
+        else
+        {
+            $Invocation.InvocationName.Substring(0,$Invocation.InvocationName.LastIndexOf("\"));
+        }
+    }
+
+    $Root = Get-ScriptDirectory
+
+    function Create-ShortCut {
```

Comment:
Creating shortcut is also unnecessary IMO and not core to Mobius development. It might be simpler just to stick to launching the dev prompt. Users can create a shortcut to Init script if needed.